Packs/GroupIB_ThreatIntelligenceAttribution/IncidentTypes/incidenttype-GIB_APT_Threat.json

```diff
@@ -0,0 +1,46 @@
+{
+	"autorun": true,
+	"cacheVersn": 0,
+	"closureScript": "",
+	"color": "#AA00FF",
+	"days": 0,
+	"daysR": 0,
+	"default": false,
+	"definitionId": "",
+	"detached": false,
+	"disabled": false,
+	"extractSettings": {
+		"fieldCliNameToExtractSettings": {
+			"gibrelatedindicatorsdata": {
+				"extractAsIsIndicatorTypeId": "",
+				"extractIndicatorTypesIDs": [],
+				"isExtractingAllIndicatorTypes": true
+			}
+		},
+		"mode": "Specific"
+	},
+	"fromServerVersion": "",
+	"hours": 0,
+	"hoursR": 0,
+	"id": "GIB APT Threat",
+	"itemVersion": "",
+	"layout": "GIB APT Threat Layout",
+	"locked": false,
+	"name": "GIB APT Threat",
+	"onChangeRepAlg": 0,
+	"packID": "GroupIB_ThreatIntelligenceAttribution",
+	"packName": "",
+	"packPropagationLabels": [
+		"all"
+	],
+	"playbookId": "Incident Postprocessing - Group-IB Threat Intelligence \u0026 Attribution",
```

Comment:
I noticed all the incident fields have this encoding. Is this how it was exported? Was it changed by the SDK?
I suspect this will cause issues. If this is not intended let me know and I'll look into it.

Reply:
Yes, it was so amortized from cortex xsoar and has not yet caused any problems for us at this time.
The playbook ID itself is Incident Postprocessing - Group-IB Threat Intelligence & Attribution

Should I change in all updates \u0026 to & ?

Reply:
Hi @idovandijk , can you please follow up?

Reply:
Hey @Kchekh, looking into it. I'll let you know as soon as I get a reply.
In the meantime, can you verify that this incident type uploads successfully and gets the correct playbook association when imported to your XSOAR server?

Reply:
I checked this now, yes the playbook binds to incidents and executes normally, in the incident table in the Playbook column it shows up with normal coding (Incident Postprocessing - Group-IB Threat Intelligence & Attribution)

Reply:
Great, I've also verified this on my tenant. I'm approving this from my side!